packages/netlify-cms-core/src/actions/config.js

```diff
@@ -111,12 +106,15 @@ export function loadConfig() {
     try {
       const preloadedConfig = getState().config;
       const configUrl = getConfigUrl();
-      const loadedConfig = await getConfig(configUrl, preloadedConfig && preloadedConfig.size > 1);
+      const loadedConfig =
+        bootstrapConfig && bootstrapConfig.loadConfig === false
```

Comment:
The conditional here will default to not use the `preloadedConfig` and break my current projects.

Good idea to not try loading the `config.yml` upon conditional config though. I think it should be something like

    bootstrapConfig && bootstrapConfig.ignorePreloadConfig
          ? {}
          : await getConfig(configUrl, preloadedConfig && preloadedConfig.size > 1);

cc/ @erquhart

Reply:
I think that will work.